docs/user/cropping-and-transforming.md

```diff
@@ -25,8 +25,6 @@ page3.mediabox.upper_right = (
 writer.add_page(page3)
 
 # Add some JavaScript to launch the print window on opening this PDF.
-# The password dialog may prevent the print dialog from being shown,
```

Comment:
Why can we remove this?

Reply:
This JavaScript distracts from the example, and I wanted to minimize it. Could be removed but is the only `add_js` example in the documentation.

Reply:
Please either keep this here or add a dedicated example at an appropriate location instead.

Reply:
Reverted.